ui/js/controllers/perf/alerts.js

```diff
@@ -529,6 +529,24 @@ perf.controller('AlertsCtrl', [
                 });
         };
 
+        $scope.summaryTitle = {
+            html: '<i class="fa fa-spinner" aria-hidden="true"></i>',
+            promise: null
+        };
+
+        $scope.getSummaryTitle = function(id) {
+            $scope.summaryTitle.promise = PhAlerts.getAlertTitle(id);
+            $scope.summaryTitle.promise.then(
+                function(summaryTitle) {
+                    $scope.summaryTitle.html = '<p>' + summaryTitle + '</p>';
+                });
+        };
+
+        $scope.resetSummaryTitle = function() {
+            $scope.summaryTitle.promise.cancel();
+            $scope.summaryTitle.html = '<i class="fa fa-spinner" aria-hidden="true"></i>';
```

Comment:
You missed a spot where we needed fa-spin. But actually this should have been fa-pulse. I fixed this for you and merged the PR, since everything else looked good.